scripts/run.sh

```diff
@@ -3,10 +3,10 @@ set -e
 
 # e.g.,
 #
-# run without e2e tests
+# run a 5 node network with the avalanche-network-runner
```

Comment:
TODO: finish refactoring this after avalanchego v1.9.6 release is out (this script does not work when running with an rc where there is no signed binary available ie v1.9.6-rc.0)